examples/app-settings/.eslintrc.cjs

```diff
@@ -5,6 +5,7 @@ module.exports = {
             version: 'detect',
         },
     },
+    ignorePatterns: ['**/*.md'],
```

Comment:
I think eslint doesn't check md files by default.

Reply:
hm, i was only able to fix the linting issues that were generating by ignoring the md files in eslintrc